java-openrewrite/openrewrite.batch.yaml

```diff
@@ -0,0 +1,42 @@
+name: run-openrewrite
+description: |
+  This batch change adds the [Rewrite](https://github.com/openrewrite/rewrite)
+  plugin to `build.gradle` files along with the "AutoFormat" recipe. It then
+  executes the rewrite.
+
+on:
+  - repositoriesMatchingQuery: repo:github.com/openrewrite/spring-petclinic-migration file:build.gradle fork:yes
+
+steps:
+  # Use comby to rewrite all build.gradle files found by the search query above
+  # to include the "org.openrewrite.rewrite" plugin.
+  - run: |
+      comby -in-place \
```

Comment:
Would that also add it if it was already there?

Reply:
Yeah, that's the downside of this. There's ways around this, but I haven't found one that doesn't scare readers away. I think as an example this is okay. But I do want to find a way to make this general pattern of "add this to this file in case it's not in there yet" easy. I'll try to see if comby's rewrite rules can do something like this